src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -577,6 +577,7 @@ func (c *controller) UploadLogs(ctx context.Context, wg *sync.WaitGroup, status
 	c.log.Infof("Start sending logs")
 	podName := ""
 	ticker := time.NewTicker(LogsUploadPeriod)
+	var errorDetected bool = false
```

Comment:
`errorDetected` is very generic. Maybe `isControllerErrorDetected`?

Reply:
It was meant to be generic because at this point I do not know what is the source of the error. And it is not a controller error but an error in the cluster state. Therefore I think the short version is better and was designed to be generic.

Reply:
So maybe let's change it to `clusterErrorDetected`? keeping it as an "error" can mean there was an error even in this function. I would like it to refer to where the error is coming from, even if it can only eliminate some trivial places.

Reply:
logic has changed. errorDetected var has been removed